### DISCUSSION TITLE
Timed out after waiting 4000ms for your remote page to load with NextJS Image tag

### DISCUSSION THREAD

**Discussion by softshipper** [vercel/next.js#24909] [Help]
Hi all 
Running E2E with Cypress on NextJS application I have got the following error message:
```
CypressError
Timed out after waiting 4000ms for your remote page to load.

Your page did not fire its load event within 4000ms.

You can try increasing the pageLoadTimeout value in cypress.json to wait longer.

Browsers will not fire the load event until all stylesheets and scripts are done downloading.

When this load event occurs, Cypress will continue running commands.

Because this error occurred during a before each hook we are skipping the remaining tests in the current suite: E2E testing /
```
As you can see on the picture below, it seems that the image does not get loaded:

![2021-05-08_09-45](https://user-images.githubusercontent.com/56756949/117531317-3dd87e00-afe2-11eb-8ca7-693ea614a382.png)

What am I doing wrong? Why the image does not get loaded correctly? 
Below is the source code.
[cynext.zip](https://github.com/cypress-io/cypress/files/6445453/cynext.zip)  

Thanks in advanced

**Comment by bobarros** [comment]
I had no trouble with your code. 
My best guess? You're trying to run a cypress test without running your app. Do you see I have two terminals, one running your we application and the other with Cypress?

![cypress](https://user-images.githubusercontent.com/34839745/117536464-889dca00-afd1-11eb-9cf1-1b7693d067f7.png)

Ps 1: it's not a good practice to upload zip files. You're in git hub, you might share a repo with your code. 
Ps 2: remove .next directory before sharing your repo.
PS 3: Whenever asking for help, describe all the steps you're doing it. Without a good look at your code, I had no idea what you were trying to do.  Most people don't even bother to help in these cases, and you will never know why.

**Reply by softshipper** [reply]
Hi @bobarros 
First of all thanks for your answer. As you can see in the picture below:
![image](https://user-images.githubusercontent.com/56756949/117537258-0678ca00-b000-11eb-8c56-9b8684a78ed4.png) 

The app is running and still receives the error message `Timed out after waiting 4000ms for your remote page to load.`.

Consider the addresses they are the same. Maybe it should not be the same. 


I am going to remember.
> Ps 1: it's not a good practice to upload zip files. You're in git hub, you might share a repo with your code.
> Ps 2: remove .next directory before sharing your repo.
> PS 3: Whenever asking for help, describe all the steps you're doing it. Without a good look at your code, I had no idea what you were trying to do. Most people don't even bother to help in these cases, and you will never know why.

Best regards

**Reply by softshipper** [reply]
Hi @bobarros
I have found something interesting.
![Screenshot from 2021-05-08 13-37-06](https://user-images.githubusercontent.com/56756949/117537736-35dd0600-b003-11eb-82b1-956d1467b6bf.png)
Running the app on the right-hand side and at the same time running the test on cypress I have never got any error message. 
This is really weired.

**Reply by bobarros** [reply]
I would love to believe that you got the first right-handed Cypress ever hahaha

One screenshot it's hitting port 3000 and the other 8080 though

**Reply by softshipper** [reply]
@bobarros 
I have tested with a different port. That is why you see two different ports. Can you tell me, what am I doing wrong?

**Comment by softshipper** [comment]
I have recorded a video to get a better understand:


https://user-images.githubusercontent.com/56756949/117549030-802caa00-b038-11eb-9daf-9e62045f43fa.mp4

**Reply by bobarros** [reply]
It's hard to know.  Even with the video. 
If you don't realize by Monday, send me an email, we can do a video call or something with your shared screen and I see what I can do
